doc/source/guide/introduction/getting_started.rst

```diff
@@ -2,4 +2,288 @@
+   kernel = odl.phantom.cuboid(space, [-0.05, -0.05], [0.05, 0.05])
+   conv = Convolution(kernel)
+   phantom = odl.phantom.shepp_logan(space, modified=True)
+   data = conv(phantom)
+   
+   kernel.show('kernel')
+   phantom.show('phantom')
+   data.show('convolved phantom')
+
+.. image:: figures/getting_started_kernel.png
+
+.. image:: figures/getting_started_phantom.png
+
+.. image:: figures/getting_started_convolved.png
+
+We can use this data to try to solve the inverse problem. To solve that, we need further properties of the operator. Specifically, we need an `Operator.adjoint`. Luckily, the convolution operator is self adjoint if the kernel is symmetric, so we can add: 
+
+.. code-block:: python
+
+   class Convolution(odl.Operator):
+       ...  # old code
+
+       @property  # making the adjoint a property lets users access it as conv.adjoint
+       def adjoint(self):
+           return self  # the adjoint is the same as the operator
+   
+Now lets try with the simple `landweber` solver:
+
+.. code-block:: python
+
+   # Need operator norm for step length (omega)
+   opnorm = odl.power_method_opnorm(conv, niter=10)
+   
+   x = space.zero()
+   odl.solvers.landweber(conv, x, data, niter=100, omega=1/opnorm**2)
+   x.show('landweber solution')
+   
+.. image:: figures/getting_started_landweber.png
+
+This solution is not very good, mostly due to the ill-posedness of the convolution operator. 
+Other solvers, like the `conjugate_gradient` solver give similar results:
+
+.. code-block:: python
+
+   x = space.zero()
+   odl.solvers.conjugate_gradient(conv, x, data, niter=100)
+   x.show('conjugate gradient solution')
+
+.. image:: figures/getting_started_conjugate_gradient.png
+
+A method to remedy this problem is to instead consider a regularized problem. One of the classic regularizers is `Tikhonov regularization <https://en.wikipedia.org/wiki/Tikhonov_regularization>`_ where we instead study the problem
+
+.. math::
+   
+   \min_f ||Af - g||_2 + ||Bf||_2
+
+Where :math:`B` is a regularizer. To solve this with the above solvers, we can find the first order optimality conditions
+
+.. math::
+ 
+   2 A^* (Af - g) + 2 B^* B f =0
+
+Where :math:`A^*` is the adjoint of :math:`A`. This can be rewritten on the form :math:`Ax=b`:
+
+.. math::
+
+   (A^* A + B^* B) f = A^* g
+
+We can try solving this with :math:`B` as the `IdentityOperator` using ODL:
+
+.. code-block:: python
+
+   I = odl.IdentityOperator(space)
+   reg_op = conv.adjoint * conv + 0.1 * I.adjoint * I
+   rhs = conv.adjoint(data)
+   
+   x = space.zero()
+   odl.solvers.conjugate_gradient(reg_op, x, rhs, niter=100)
+   x.show('Tikhonov identity conjugate gradient solution')
+
+.. image:: figures/getting_started_tikhonov_identity_conjugate_gradient.png
+
+Slightly better, but no major upgrade. What about letting :math:`B` be the `Gradient`?
+
+.. code-block:: python
+
+   grad = odl.Gradient(space)
+   reg_op = conv.adjoint * conv + 0.0001 * grad.adjoint * grad
```

Comment:
The different scaling should not go uncommented. Perhaps it's easier to include a (nonnegative) parameter in the goal functional, i.e.
```
||Af - g||^2 + a^2 ||Bf||^2
```
That would result in `A^* A + a * B^* B` in optimality and normal equations, probably that is easier to understand.